portal/portal-util/util/src/java/org/sakaiproject/portal/util/CSSUtils.java

```diff
@@ -224,8 +255,12 @@ public static String getCssToolSkinLink(String skin, boolean isInlineRequest) {
 		String headCssPrintSkin = "<link href=\"" 
 				+ getCssPrintSkinCDN(skin)
 				+ "\" type=\"text/css\" rel=\"stylesheet\" media=\"print\" />\n";
+
+		String headCssPropertiesSkin = "<link href=\"" 
+				+ getCssPropertiesSkinCDN(skin)
+				+ "\" type=\"text/css\" rel=\"stylesheet\" media=\"screen, tty, tv, handheld, projection\" />\n";
 		
-		return headCssToolSkin + headCssPrintSkin;
+		return headCssToolSkin + headCssPrintSkin + headCssPropertiesSkin;
```

Comment:
@ern this is what we were going to ask about during ui call today. I wrote this with the rationale that all the other CSS files generated in library have these functions, so the new properties.css file should have them as well even though there's not a use case for using that file currently outside of ckeditor. I worried that someone would come along and expect to be able to call that properties.css file the same way they do tool.css and would get tripped up if these functions didn't exist.